CHANGELOG.md

```diff
@@ -40,6 +39,7 @@
   * `-query-scheduler.ring.etcd.*`
   * `-overrides-exporter.ring.etcd.*`
 * [FEATURE] Distributor, ingester, querier, query-frontend, store-gateway: add experimental support for native histograms. Requires that the experimental protobuf query result response format is enabled by `-query-frontend.query-result-response-format=protobuf` on the query frontend. #4286 #4352 #4354 #4376 #4377 #4387 #4396 #4425 #4442 #4494 #4512 #4513 #4526
+* [ENHANCEMENT] Add timezone information to Alpine Docker images. #4583 
```

Comment:
One last fix to make the linter happy :facepalm: 
```suggestion
* [ENHANCEMENT] Add timezone information to Alpine Docker images. #4583
```

Reply:
let's hope this does it 🤞